src/azure-cli-core/azure/cli/core/telemetry.py

```diff
@@ -307,14 +334,19 @@ def set_raw_command_name(command):
 
 @decorators.suppress_all_exceptions(raise_in_diagnostics=True)
 def add_extension_event(extension_name, properties, instrumentation_key=DEFAULT_INSTRUMENTATION_KEY):
-    # Inject correlation ID into the extension event
+    set_custom_properties(properties, 'ExtensionName', extension_name)
+    add_event("extension", properties, instrumentation_key=instrumentation_key)
```

Comment:
Strings should be surrounded by a pair of single quotes instead of double quotes